src/core/api/index.js

```diff
@@ -113,3 +113,12 @@ export function fetchProfile({ api }) {
     state: api,
   });
 }
+
+export function featured({ addonType, api }) {
+  return callApi({
+    endpoint: 'addons/featured',
+    params: { app: api.clientApp, type: addonType },
+    schema: addon,
```

Comment:
Shouldn't this be similar to [search](https://github.com/mozilla/addons-frontend/blob/6eed6fd46d1d5cde08fd46a104d82e8696f846e0/src/core/api/index.js#L66) where it's a list of add-ons?

```
 schema: { results: arrayOf(addon) },